src/msg/async/EventKqueue.cc

```diff
@@ -22,34 +22,129 @@
 #undef dout_prefix
 #define dout_prefix *_dout << "KqueueDriver."
 
+#define KEVENT_NOWAIT 0
```

Comment:
@yuyuyu101 
Leftover from previous code. Usually I replace hard values with descriptive constants.
I'll see if it still fits the purpose.